src/store/model/microcenter.ts

```diff
@@ -32,11 +33,65 @@ const microCenterLocationToId: Map<string, string> = new Map([
 	['yonkers', '105']
 ]);
 
+let links: Link[] = [];
 let storeId: string;
-if (microCenterLocationToId.get(microCenterLocation) === undefined) {
-	storeId = '029';
-} else {
-	storeId = microCenterLocationToId.get(microCenterLocation)!;
+for (const microCenterLocation of microCenterLocations) {
+	if (microCenterLocationToId.get(microCenterLocation) === undefined) {
+		const logString = `No MicroCenter location named ${microCenterLocation}, skipping.`;
+		logger.warn(logString);
+	} else {
+		storeId = microCenterLocationToId.get(microCenterLocation)!;
```

Comment:
What does the `!` mean here?

Reply:
Non-null assertion. It was there before I moved this line, not sure if it's still needed.